### PR TITLE
Add Cipher article

### PR DESCRIPTION
**Pull request by salarson**
#SignOff

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/prepare-tls-1.2-in-office-365.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ If you are using any on-premises infrastructure for hybrid scenarios or Active D
 The following resources provide guidance to help make sure that your clients are using TLS 1.2 or a later version and to disable TLS 1.0 and 1.1.
 
 - For Windows 7 clients that connect to Office 365, make sure that TLS 1.2 is the default secure protocol in WinHTTP in Windows. For more information see [KB 3140245 - Update to enable TLS 1.1 and TLS 1.2 as a default secure protocols in WinHTTP in Windows](https://support.microsoft.com/help/3140245/update-to-enable-tls-1-1-and-tls-1-2-as-a-default-secure-protocols-in).
+- [TLS cipher suites supported by Office 365](https://docs.microsoft.com/en-us/microsoft-365/compliance/technical-reference-details-about-encryption?view=o365-worldwide#tls-cipher-suites-supported-by-office-365)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/prepare-tls-1.2-in-office-365.md]:
```suggestion
- [TLS cipher suites supported by Office 365](/microsoft-365/compliance/technical-reference-details-about-encryption?view=o365-worldwide#tls-cipher-suites-supported-by-office-365)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please change to relative link as indicated.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Pull request merged**